spectacles/cli.py

```diff
@@ -375,23 +377,24 @@ def _build_validator_subparser(
         action=EnvVarAction,
         env_var="LOOKER_GIT_BRANCH",
         required=True,
-        help="The branch of your project that spectacles will use to run queries.",
+        help="The branch of your project that Spectacles will use to run queries.",
     )
-    base_subparser.add_argument(
+    group = base_subparser.add_mutually_exclusive_group()
+    group.add_argument(
         "--remote-reset",
         action=EnvVarStoreTrueAction,
         env_var="SPECTACLES_REMOTE_RESET",
         help="When set to true, the SQL validator will tell Looker to reset the \
             user's branch to the revision of the branch that is on the remote. \
             WARNING: This will delete any uncommited changes in the user's workspace.",
     )
-    base_subparser.add_argument(
-        "--import-projects",
```

Comment:
This argument was deleted somehow (maybe during rebasing?) and needs to be restored. Spectacles won't run with this PR at the moment.